doc/source/reference/arrays.rst

```diff
@@ -19,20 +19,21 @@ objects contained with a :class:`Index`, :class:`Series`, or
 For some data types, pandas extends NumPy's type system. String aliases for these types
 can be found at :ref:`basics.dtypes`.
 
-=================== ========================= ============================= =============================
-Kind of Data        pandas Data Type          Scalar                        Array
-=================== ========================= ============================= =============================
-TZ-aware datetime   :class:`DatetimeTZDtype`  :class:`Timestamp`            :ref:`api.arrays.datetime`
-Timedeltas          (none)                    :class:`Timedelta`            :ref:`api.arrays.timedelta`
-Period (time spans) :class:`PeriodDtype`      :class:`Period`               :ref:`api.arrays.period`
-Intervals           :class:`IntervalDtype`    :class:`Interval`             :ref:`api.arrays.interval`
-Nullable Integer    :class:`Int64Dtype`, ...  (none)                        :ref:`api.arrays.integer_na`
-Categorical         :class:`CategoricalDtype` (none)                        :ref:`api.arrays.categorical`
-Sparse              :class:`SparseDtype`      (none)                        :ref:`api.arrays.sparse`
-Strings             :class:`StringDtype`      :class:`str`                  :ref:`api.arrays.string`
-Boolean (with NA)   :class:`BooleanDtype`     :class:`bool`                 :ref:`api.arrays.bool`
-PyArrow             :class:`ArrowDtype`       Python Scalars or :class:`NA` :ref:`api.arrays.arrow`
-=================== ========================= ============================= =============================
+=================== ========================== ============================= =============================
+Kind of Data        pandas Data Type           Scalar                        Array
+=================== ========================== ============================= =============================
+TZ-aware datetime   :class:`DatetimeTZDtype`   :class:`Timestamp`            :ref:`api.arrays.datetime`
+Timedeltas          (none)                     :class:`Timedelta`            :ref:`api.arrays.timedelta`
+Period (time spans) :class:`PeriodDtype`       :class:`Period`               :ref:`api.arrays.period`
+Intervals           :class:`IntervalDtype`     :class:`Interval`             :ref:`api.arrays.interval`
+Nullable Integer    :class:`Int64Dtype`, ...   (none)                        :ref:`api.arrays.integer_na`
+Nullable Float      :class:`Float64Dtype`, ... (none)                        :ref:`api.arrays.float_na`
```

Comment:
Correct, since ``` :class:`Float64Dtype`, ... ``` was wider than the section boundaries (`===...`) everything needed to shift.

Also renamed `Boolean (with NA)` to `Nullable Boolean` to match the int and float titles.